src/content/docs/en/guides/typescript.mdx

```diff
@@ -122,11 +122,11 @@ import Layout from '@layouts/Layout.astro';
 You may want to add a property to the global object. You can do this by adding top-level declarations using the `declare` keyword to your `env.d.ts` file:
 
 ```ts title="env.d.ts"
-declare const myString: string;
+declare var myString: string;
 declare function myFunction(): boolean;
 ```
 
-This will provide typing to `globalThis.myString` and `globalThis.myFunction`, as well as `window.myString` and `window.myFunction`.
+This will provide typing to `globalThis.myString` and `globalThis.myFunction`, as well as `window.myString` and `window.myFunction`. Make sure to use `var` instead of `const` for type declarations.
```

Comment:
```suggestion
This will provide typing to `globalThis.myString` and `globalThis.myFunction`, as well as `window.myString` and `window.myFunction`.
```
If we incorporate the information earlier, then we don't need a "make sure" comment!

Reply:
It's okay to remove it, even though we didn't incorporate the change from above.